components/expenses/ExpenseForm.js

```diff
@@ -749,7 +749,7 @@ const ExpenseFormBody = ({
       )}
       {step === EXPENSE_FORM_STEPS.EXPENSE && (
         <StyledCard mt={4} p={[16, 24, 32]} overflow="initial">
-          <ExpensePayeeDetails expense={formik.values} host={collective.host} borderless collective={collective} />
+          <ExpenseSummaryAdditionalInformation expense={formik.values} host={collective.host} collective={collective} />
```

Comment:
Renamed the component as the legacy name was misleading, this holds more info than just the payee details